ktor-client/ktor-client-cio/jvm/test/io/ktor/client/engine/cio/CIOHttpsTest.kt

```diff
@@ -110,8 +113,15 @@ class CIOHttpsTest : TestWithKtor() {
                 }
 
                 test { client ->
-                    val actual = client.get<String>("https://127.0.0.1:$serverPort/")
-                    assertEquals("Hello, world", actual)
+                    try {
+                        println("Starting: ${suite.name}")
```

Comment:
println

Reply:
The test could freeze, so debug print is helpful.
Maybe we should keep it?